typebackend/typingviews.py

```diff
@@ -73,3 +68,24 @@ def post(self,request):
             return Response({'success':True})
         else:
             return Response({'success':False,'error':serializers.errors},status=status.HTTP_400_BAD_REQUEST)
+
+class Graphdata(APIView):
+    permission_classes=[IsAuthenticated]
+
+    def get(self,request,days):  
+        date_typed_log={}
+
+        userlog=PractiseLog.objects.filter(user=request.user,taken_at__gte=timezone.now()-datetime.timedelta(days=int(days)))
```

Comment:
What if **days** query param is not passed?
Add a default value

Reply:
It will throw error !

Reply:
yeah add a default value in function signature